src/plugins/dashboard/public/application/actions/unlink_from_library_action.tsx

```diff
@@ -40,14 +41,14 @@ export class UnlinkFromLibraryAction implements ActionByType<typeof ACTION_UNLIN
   public readonly id = ACTION_UNLINK_FROM_LIBRARY;
   public order = 15;
 
-  constructor() {}
+  constructor(private toasts: NotificationsStart['toasts']) {}
```

Comment:
nit:
Would be great to have from the start something like: 
```
private deps: {toasts: NotificationsStart['toasts']}
```

would be annoying to update when we will need new dependency  

(same for link action)

Reply:
Fixed both instances of this. Makes sense.